$MirOS$

	Raise along with FULLPKGNAME-nssckbi in Makefile
	when ../files/certdata.c changes.

--- security/nss/lib/ckfw/builtins/nssckbi.h.orig	Thu May 21 19:50:05 2009
+++ security/nss/lib/ckfw/builtins/nssckbi.h	Mon Aug  3 18:50:50 2009
@@ -76,7 +76,7 @@
  */
 #define NSS_BUILTINS_LIBRARY_VERSION_MAJOR 1
 #define NSS_BUILTINS_LIBRARY_VERSION_MINOR 75
-#define NSS_BUILTINS_LIBRARY_VERSION "1.75"
+#define NSS_BUILTINS_LIBRARY_VERSION "1.75.0.1"
 
 /* These version numbers detail the semantic changes to the ckfw engine. */
 #define NSS_BUILTINS_HARDWARE_VERSION_MAJOR 1
